core/src/main/java/com/sequenceiq/cloudbreak/service/clustertemplate/ClusterTemplateService.java

```diff
@@ -52,11 +53,16 @@ public ClusterTemplate getByName(String name, IdentityUser user) {
         return clusterTemplateRepository.findByNameInAccount(name, user.getAccount(), user.getUserId());
     }
 
-    public ClusterTemplate create(IdentityUser user, ClusterTemplate clusterTemplate) {
+    public ClusterTemplate create(IdentityUser user, ClusterTemplate clusterTemplate, Organization organization) {
         LOGGER.debug("Creating clusterTemplate: [User: '{}', Account: '{}']", user.getUsername(), user.getAccount());
         ClusterTemplate savedClusterTemplate;
         clusterTemplate.setOwner(user.getUserId());
         clusterTemplate.setAccount(user.getAccount());
+        if (organization != null) {
```

Comment:
Code duplication

Reply:
Cannot remove duplication until it's not an OrganizationAwareResource. I will create a util service and move this logic there once these resources will implement the OrganizationAwareResource interface. It will happen when auth is implemented for them.